tests/lmeval/configs/vl_fp8_dynamic_per_token.yaml

```diff
@@ -10,7 +10,6 @@ lmeval:
     convert_img_format: True
   task: mmmu_val_economics
   num_fewshot: 0
-  limit: 1000
```

Comment:
are there > 1000 samples here?

Reply:
the default is 1000, so removing this doesn't change the config passed to lm eval. the mmmu economics dataset doesn't have more than 1000 samples though, so this is just using all of them